tools/semibin/semibin.xml

```diff
@@ -158,13 +158,13 @@ ls output
             <section name="bin">
                 <param name="max_node" value="1"/>
                 <param name="max_edges" value="200"/>
-                <param name="minfasta_kbs" value="200"/>
+                <param name="minfasta_kbs" value="2"/>
                 <param name="no_recluster" value="false"/>
             </section>
             <param name="extra_output" value="data,coverage,contigs"/>
-            <output_collection name="output_recluster_bins" count="0"/>
-            <output_collection name="output_bins" count="3">
-                <element name="0" ftype="fasta">
+            <output_collection name="output_recluster_bins" count="39"/>
```

Comment:
Could you add tests for the elements in the collection? Similar to the ones you commented below. Thanks